integration-tests/chaos/chaos_test.go

```diff
@@ -27,6 +28,8 @@ import (
 
 var (
 	defaultOCRSettings = map[string]interface{}{
+		"toml": client.AddNetworksConfig(`[OCR]
+Enabled = true`, networks.SelectedNetwork),
```

Comment:
This equivalent format is also available for one-liners:
```suggestion
		"toml": client.AddNetworksConfig(`OCR.Enabled = true`, networks.SelectedNetwork),
```